modules/odc-monitoring-your-project-metrics.adoc

```diff
@@ -10,7 +10,7 @@ After you create applications in your project and deploy them, you can use the *
 
 .Procedure
 
-. On the left navigation panel of the *Developer* perspective, click *Observe* to see the *Dashboard*, *Metrics*, *Alerts*, and *Events* for your project.
+. Navigate to *Observe* to see the *Dashboard*, *Metrics*, *Alerts*, and *Events* for your project.
```

Comment:
[The ISG guidance](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#n) is to avoid 'navigate' in favor of alternatives such as "go to":